src/caencoder.c

```diff
@@ -976,53 +976,51 @@ static int ca_encoder_get_entry_data(CaEncoder *e, CaEncoderNode *n) {
         /* Note that any follow-up structures from here are unaligned in memory! */
 
         if (child->stat.st_uid == e->cached_uid && e->cached_user_name) {
-                p = mempcpy(p,
-                            &(CaFormatHeader) {
-                                    .type = htole64(CA_FORMAT_USER),
-                                    .size = htole64(offsetof(CaFormatUser, name) + strlen(e->cached_user_name) + 1),
-                            },
-                            sizeof(CaFormatHeader));
+                CaFormatHeader header = {
```

Comment:
I don't think that'd help much,, given that the argument to `mempcpy()` is already const. But I can patch this in after the fact.